doc/classes/DTLSServer.xml

```diff
@@ -34,9 +35,55 @@
 		            while p.get_available_packet_count() &gt; 0:
 		                print("Received message from client: %s" % p.get_packet().get_string_from_utf8())
 		                p.put_packet("Hello DTLS client".to_utf8())
-		[/codeblock]
-		[codeblock]
-		# client.gd
+		[/gdscript]
+		[csharp]
+		using Godot;
+		using System;
+		// ServerNode.cs
+		public class ServerNode : Node
+		{
+		    public DTLSServer Dtls = new DTLSServer();
+		    public UDPServer Server = new UDPServer();
+		    public Godot.Collections.Array&lt;PacketPeerDTLS&gt; Peers = new Godot.Collections.Array&lt;PacketPeerDTLS&gt;();
+		    public override void _Ready()
+		    {
+		        Server.Listen(4242);
+		        var key = GD.Load&lt;CryptoKey&gt;("key.key"); // Your private key.
+		        var cert = GD.Load&lt;X509Certificate&gt;("cert.crt"); // Your X509 certificate.
+		        Dtls.Setup(key, cert);
+		    }
+		    public override void _Process(float delta)
+		    {
+		        while (Server.IsConnectionAvailable())
+		        {
+		            PacketPeerUDP peer = Server.TakeConnection();
+		            PacketPeerDTLS dtlsPeer = Dtls.TakeConnection(peer);
+		            if (dtlsPeer.GetStatus() != PacketPeerDTLS.Status.Handshaking)
+		            {
+		                continue; // It is normal that 50% of the connections fails due to cookie exchange.
+		            }
+		            GD.Print("Peer connected!");
+		            Peers.Add(dtlsPeer);
+		        }
+		        foreach (var p in Peers)
```

Comment:
Spacing after blocks is not a general rule, but here it makes sense due to the size of the blocks.

```suggestion
		        }

		        foreach (var p in Peers)
```